apps/screenshot-diff/src/util/image-diff/test/e2e.test.ts

```diff
@@ -122,4 +122,16 @@ describe("e2e", () => {
 
     expect(difference).toMatchSnapshot();
   });
+
+  it("find differences on similar images", async () => {
```

Comment:
completely different you mean? If the result value is 1 ?

Reply:
Similar for humans and for odiff. But you are wright, I can find a better name